build/version.props

```diff
@@ -0,0 +1,7 @@
+<!-- This file may be overwritten by automation. Only values allowed here are VersionPrefix and VersionSuffix.  -->
+<Project>
+  <PropertyGroup>
+    <VersionPrefix>5.0.0</VersionPrefix>
```

Comment:
We should bump this to 6 to match the appveyor version (I think I need to do this in our other repos as well)